### PR TITLE
Order containers

### PR DESCRIPTION
**Pull request by keharper**
This PR adds documentation for containers in the order drop-in.

### CODE REVIEW

**Review by KostiantynFandeliuk — APPROVED** [review]

File: src/content/docs/dropins/order/containers/customer-details.mdx [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+        ['orderData', 'OrderDataModel', 'No', 'A structured object containing transformed order data. It can be used as an initial value if data is not fetched from the backend, serving as a fallback.'],
+        ['title', 'string', 'No', 'Enables setting a custom title to replace the default one during container interaction.'],
+        ['className', 'string', 'No', 'Allows custom CSS classes to be applied to the form.'],
+        ['slots.OrderReturnInformation', 'SlotProps', 'No', 'Allows adding or expanding information return details section by including additional data or attributes.'],
+    ]}
+/>
+
+## Example
+
+The following example demonstrates how to integrate the CustomerDetails container:
```
Comment (bdenham) [inline src/content/docs/dropins/order/containers/customer-details.mdx]:
```suggestion
The following example demonstrates how to integrate the `CustomerDetails` container:
```

File: src/content/docs/dropins/order/containers/customer-details.mdx [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+
+The return details section is available exclusively on return pages. It provides information about the return.
+
+<Diagram caption="CustomerDetails container">
+  ![CustomerDetails  container](@images/dropins/order/customer-details.png)
+</Diagram>
+
+## Configurations
+
+The CustomerDetails container provides the following configuration options:
```
Comment (bdenham) [inline src/content/docs/dropins/order/containers/customer-details.mdx]:
```suggestion
The `CustomerDetails` container provides the following configuration options:
```

File: src/content/docs/dropins/order/containers/customer-details.mdx (hunk 1 above)
Comment (bdenham) [inline src/content/docs/dropins/order/containers/customer-details.mdx]:
```suggestion
The following example demonstrates how to integrate the `CustomerDetails` container:
```

File: src/content/docs/dropins/order/containers/order-cancel-form.mdx [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+        ['orderRef', 'string', 'Yes', 'ID of the order to be canceled.'],
+        ['pickerProps', 'PickerProps', 'No', 'Configuration for the picker used to display and select the reason for order cancellation.'],
+        ['submitButtonProps', 'ButtonProps', 'No', 'Configuration for the button used to submit the order cancellation.'],
+        ['cancelReasons', 'PickerOption[]', 'Yes', 'An array of reasons available for order cancellation.'],
+    ]}
+/>
+
+## Example
+
+The `OrderCancelForm` container is not directly integrated within the boilerplate, but it is delivered as part of the `OrderStatus` container. However, the OrderCancelForm container can also be used independently to create custom implementations.
```
Comment (bdenham) [inline src/content/docs/dropins/order/containers/order-cancel-form.mdx]:
```suggestion
The `OrderCancelForm` container is not directly integrated within the boilerplate, but it is delivered as part of the `OrderStatus` container. However, the `OrderCancelForm` container can also be used independently to create custom implementations.
```

File: src/content/docs/dropins/order/containers/order-cancel-form.mdx [hunk 4]
```diff
@@ -0,0 +1,46 @@
...
+
+Here’s an integration example from the drop-in development environment:
```
Comment (bdenham) [inline src/content/docs/dropins/order/containers/order-cancel-form.mdx]:
```suggestion
Here’s an integration example from the drop-in component development environment:
```

File: src/content/docs/dropins/order/containers/order-cost-summary.mdx [hunk 5]
```diff
@@ -0,0 +1,51 @@
...
+description: Learn about the OrderCostSummary container.
+---
+
+import Diagram from '@components/Diagram.astro';
+import Callouts from '@components/Callouts.astro';
+import OptionsTable from '@components/OptionsTable.astro';
+import Aside from '@components/Aside.astro';
+import { Steps } from '@astrojs/starlight/components';
+
+The `OrderCostSummary` container is used on Order Details and Return Details pages to display a breakdown of order costs. It includes the following sections:
```
Comment (bdenham) [inline src/content/docs/dropins/order/containers/order-cost-summary.mdx]:
```suggestion
The `OrderCostSummary` container displays detailed order costs on the Order Details and Return Details pages. It includes the following sections:
```

File: src/content/docs/dropins/order/containers/order-cost-summary.mdx [hunk 6]
```diff
@@ -0,0 +1,51 @@
...
+- **Subtotal**: Displays the total cost of all items in the order before applying discounts, taxes, or additional charges.
+- **Shipping**: Displays the shipping cost, which depends on the shipping method, location, and weight of the order.
+- **Discount**: Displays any applicable discounts, such as promotional or volume-based offers, subtracted from the subtotal.
+- **Coupon**: Displays the value of any applied coupons and their impact on the final cost.
+- **Tax**: Displays the tax amount added to the order, calculated based on jurisdiction and item type.
+- **Total**: Displays the final payable amount, including all adjustments such as discounts, shipping, and taxes.
+
+If a value is not provided for any section (such as no discount or coupons are applied), the corresponding line is hidden. This ensures the container only displays relevant information.
+
+The display of tax amounts is configured at **Stores** > Configuration > **Sales** > **Tax** > **Order, Invoices, Credit Memos Display Settings**.
```
Comment (bdenham) [inline src/content/docs/dropins/order/containers/order-cost-summary.mdx]:
```suggestion
The settings for displaying tax amounts can be configured at **Stores** > Configuration > **Sales** > **Tax** > **Order, Invoices, Credit Memos Display Settings**.
```

File: src/content/docs/dropins/order/containers/order-cost-summary.mdx [hunk 7]
```diff
@@ -0,0 +1,51 @@
...
+
+<Diagram caption="OrderCostSummary container">
+  ![OrderCostSummary  container](@images/dropins/order/order-cost-summary.png)
+</Diagram>
+
+## Configurations
+
+The OrderCostSummary container provides the following configuration options:
```
Comment (bdenham) [inline src/content/docs/dropins/order/containers/order-cost-summary.mdx]:
```suggestion
The `OrderCostSummary` container provides the following configuration options:
```

File: src/content/docs/dropins/order/containers/order-product-list.mdx [hunk 8]
```diff
@@ -0,0 +1,45 @@
...
+
+import Diagram from '@components/Diagram.astro';
+import Callouts from '@components/Callouts.astro';
+import OptionsTable from '@components/OptionsTable.astro';
+import Aside from '@components/Aside.astro';
+import { Steps } from '@astrojs/starlight/components';
+
+The `OrderProductList` container displays a list of products associated with a specific order or return. Each item in the list is represented by a product card containing details such as the price, applied discounts, tax information, final amount, and product attributes.
+
+The display of tax amounts is configured at **Stores** > Configuration > **Sales** > **Tax** > **Order, Invoices, Credit Memos Display Settings**.
```
Comment (bdenham) [inline src/content/docs/dropins/order/containers/order-product-list.mdx]:
```suggestion
The settings for displaying tax amounts can be configured at **Stores** > Configuration > **Sales** > **Tax** > **Order, Invoices, Credit Memos Display Settings**.
```

File: src/content/docs/dropins/order/containers/order-product-list.mdx [hunk 9]
```diff
@@ -0,0 +1,45 @@
...
+
+<Diagram caption="OrderProductList container">
+  ![OrderProductList  container](@images/dropins/order/order-product-list.png)
+</Diagram>
+
+## Configurations
+
+The OrderProductList container provides the following configuration options:
```
Comment (bdenham) [inline src/content/docs/dropins/order/containers/order-product-list.mdx]:
```suggestion
The `OrderProductList` container provides the following configuration options:
```

**Review by bdenham — APPROVED** [review]
Approved pending changes.

**Pull request merged**